src/client/src/containers/ProjectNameAndOutput/index.tsx

```diff
@@ -98,6 +106,18 @@ const ProjectNameAndOutput = (props: Props) => {
     const element = e.currentTarget as HTMLInputElement;
     updateProjectName(element.value);
   };
+  const validateKey = (e: React.KeyboardEvent<HTMLInputElement>) => {
+      const element = e.target as HTMLInputElement;
+      const inputKeyCheck = /^[A-Za-z0-9_\- ]$/
+
+      if(element.value.length === 50 && inputKeyCheck.test(e.key)){
+        setprojectNameMaxLength(true);
+        e.stopPropagation();
+      }else{
```

Comment:
We should use Prettier to format these files

Reply:
woops I had it turned off